Editorial changes to Dev Fund ZIP

daira opened the pull request:
This includes three editorial changes (points 2, 4, and 7) that were suggested at https://github.com/zcash/zips/pull/324#issuecomment-586544775

It also includes other editorial changes from review with @gtank.

daira committed:
Rename the "ZF-MG" slice to "MG", and the "ZF-GU" slice to "ZF".
Clearer, and reflects common use in the forum and Helios poll.

Based on https://github.com/zcash/zips/pull/319/commits/1f851eb1c3ef75d34a042e91b6fa878d6969a63a

Co-authored-by: Eran Tromer <eran@tromer.org>
Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
Resolve a wording ambiguity: Major Grant funding decisions are intended to be subject to veto if they violate any U.S. law,
not just law relating to the ZF's reporting requirements.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
Clear up a stale reference to "both of these processes"
(originally one of the processes referred to was the Community Panel's
power to change the Funding Cap, which no longer exists).

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

acityinohio commented:
ACK/LGTM @daira, thanks!

daira committed:
Obligations under 501(c)(3) are not only reporting requirements.
This responds to one part of @tromer's point at
https://forum.zcashcommunity.com/t/final-text-of-zip-1014/36002/3

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira commented:
I made another change (https://github.com/ZcashFoundation/zips/pull/1/commits/8cfeb65925b763df019827cbe33fcf2cdb9c9356) in response to part of @tromer's comment at https://forum.zcashcommunity.com/t/final-text-of-zip-1014/36002/3 :
> [...] For example, if the MG review committee decides to award all of the MG budget to (say) lobbyists, which would conflict with ZF’s 501(c​)3 status, how would ZF proceed? Just execute the payment, report it and give up its 501(c​)3 status?

acityinohio commented:
ACK on the above @daira, that makes sense to me too

daira commented:
@zookozcash reminder that your review is needed here.

gtank commented:
We should change the title to disambiguate from ZIP-1012, which is called the same.

daira committed:
ZIP 1014: terminology.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: add ZIP numbers to references.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: cosmetics.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: minor wording fixes.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: add "for example" to sentence about ecosystem growth.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: change title to disambiguate from ZIP 1012.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014 Status to Proposed.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: wording clarification. 20% of the block subsidy is allocated from *each* block.
(The previous wording could have allowed an interpretation of any allocation that sums to 20%
over four years, which was not the intent.)

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

acityinohio commented on zip-1014.rst:
```diff
@@ -188,23 +201,23 @@ The funds SHALL be received and administered by ZF. ZF MUST disburse them as
-   judges them to violate the ZF's  reporting requirements under U.S. IRS
-   501(c)(3) or U.S. law.
+   judges them to violate U.S. law or the ZF's reporting requirements and other
+   obligations under U.S. IRS 501(c)(3).
```
Could this be changed to `other (current and future) obligations under U.S. IRS 501(c)(3)`? @zookozcash and I are working through these and we both agree to this change.

daira replied:
Yes it can, I'll make that change.

acityinohio commented on zip-1014.rst:
```diff
@@ -24,13 +24,26 @@ Terminology
 in this document are to be interpreted as described in RFC 2119. [#RFC2119]_
 
+The term "network upgrade" in this document is to be interpreted as
+described in ZIP 200. [#zip-0200]_
```
Could you also change this to say `as described in ZIP 200 and the Zcash trademark agreement?` (including this link) https://www.zfnd.org/about/contracts/2019_ECC_ZFND_TM_agreement.pdf
I understand that the term "network upgrade" has inconsistent meaning when one compares ZIP 200 to the trademark agreement, and it's something the Foundation and ECC should fix. But in the mean time I think it's important to reference both.

daira replied:
ACK.

daira replied:
That appears to be a versioned document that is subject to change. Is this the intent? :

> The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_ and the Zcash Trademark Donation and License Agreement ([#trademark]_ or successor agreement).

acityinohio replied:
Thanks @daira — do you mean that there's the addendum "or successor agreement?" Yes that is intentional, in the case that the TM agreement is superseded or updated with amendments.

acityinohio commented on zip-1014.rst:
```diff
@@ -297,22 +310,23 @@ conditions, and the prescribed use of funds, such that substantial violation,
-development and its deployment, in their activities and grant selection.
+voting and governance –either by expanding the Community Panel or a
+successor mechanism– and to integrate them into this and any other
+pertinent governance processes, by the end of 2021. ECC and ZF SHOULD place
```
Can this be changed to `and to integrate them into this process by the end of 2021`? Basically remove "and any other pertinent governance processes." We shouldn't make that edit at this stage given our attempts to minimize the number of changes to ZIP 1014, and the community can decide through future ZIPs whether this process should be used for other pertinent governance decisions.

daira replied:
ACK.

daira replied:
ACK.

acityinohio reviewed (changes requested):
Hey @zookozcash and I reviewed this live and just have a few changes we'd like to request but otherwise looks good for merging.

daira committed:
ZIP 1014: include the Zcash Trademark Donation and License Agreement as part of the definition of "network upgrade".

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: add "(current or future)" to other obligations under U.S. IRS 501(c)(3).

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira committed:
ZIP 1014: delete "and any other pertinent governance processes" from the description of process for which
it is desirable to develop "robust means of decentralized community voting and governance".

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

daira commented:
I made the requested changes and also c3dad3f1edb5cbedb10faaddee0a645577d401b1 (which is purely a clarification). Please re-review and merge into #324 .

nathan-at-least reviewed (approved):
Hello, Zooko asked me to step in temporarily to represent ECC in reviewing of these edits for both this PR and https://github.com/zcash/zips/pull/324 because he has other obligations but we want to ensure this process completes. I approve of all changes above on behalf of ECC.

acityinohio commented:
ACK, approved on the Foundation side!
